includes/REST_API/Stories_Base_Controller.php

```diff
@@ -271,6 +272,57 @@ protected function prepare_links( $post ): array {
 			];
 		}
 
+		$links = $this->add_taxonomy_links( $links, $post );
+
+		return $links;
+	}
+
+	/**
+	 * Adds a REST API links for the taxonomies.
+	 *
+	 * @since 1.12.0
+	 *
+	 * @param array   $links Links for the given post.
+	 * @param WP_Post $post Post object.
+	 *
+	 * @return array Modified list of links.
+	 */
+	private function add_taxonomy_links( array $links, WP_Post $post ): array {
```

Comment:
This is just so that any taxonomy links in the REST API use our namespace, correct?

Reply:
It is